### DISCUSSION TITLE
Account banned ? another time ...

### DISCUSSION THREAD

**Discussion by pierrealexaline** [revoxhere/duino-coin#1661] [General]
Again my account is banned ? i don't know why ? i'm in holiday, no view of what is being done with my account and/or my miners. Never i have violated the TOS voluntarily. What can i do with that ? ...i think the Duino secure process must be renewed. Because this is not the first time i'm banned. For me it's i think it's the the last time i trying duino coin. It was a great experience. But the Duino coin is not well secure. My account name is 'euskalminer'. I've only some Arduino Nano, ESP32 and other ESP models. Minor than 20 miners, and more than 40 000 duco. I am very disappointed with the banned policy.

**Comment by revoxhere** [comment]
Your account is not banned.
